userservices/client/standard_test.go

```diff
@@ -563,98 +567,100 @@ var _ = Describe("Standard", func() {
 					})
 
 					It("returns an error", func() {
-						err := standard.ValidateTargetUserPermissions(context, "request-user-id", "target-user-id", client.ViewPermissions)
-						Expect(err).To(MatchError("client: unauthorized"))
+						permissions, err := standard.GetUserPermissions(context, "request-user-id", "target-user-id")
+						Expect(err).ToNot(HaveOccurred())
+						Expect(permissions).To(BeEmpty())
 						Expect(server.ReceivedRequests()).To(HaveLen(2))
 					})
 				})
 
-				Context("with an successful response, but without the requested permission", func() {
+				Context("with an successful response with upload and view permissions", func() {
 					BeforeEach(func() {
 						server.AppendHandlers(
 							ghttp.CombineHandlers(
 								ghttp.VerifyRequest("GET", "/access/target-user-id/request-user-id"),
 								ghttp.VerifyHeaderKV("X-Tidepool-Session-Token", "test-session-token"),
 								ghttp.VerifyBody([]byte{}),
-								ghttp.RespondWith(http.StatusOK, "{\"upload\": {}}", nil)),
+								ghttp.RespondWith(http.StatusOK, "{\"upload\": {}, \"view\": {}}", nil)),
```

Comment:
Doh! Completely zoned on using string literals here. Will update all code to use string literals where there are any escaped double-quotes.